packages/simcore-sdk/src/simcore_sdk/node_ports_common/file_io_utils.py

```diff
@@ -170,21 +174,21 @@ async def download_link_to_file(
     io_log_redirect_cb: LogRedirectCB | None,
     progress_bar: ProgressBarData,
 ):
-    log.debug("Downloading from %s to %s", url, file_path)
+    _logger.debug("Downloading from %s to %s", url, file_path)
     async for attempt in AsyncRetrying(
         reraise=True,
         wait=wait_exponential(min=1, max=10),
         stop=stop_after_attempt(num_retries),
         retry=retry_if_exception_type(ClientConnectionError),
-        before_sleep=before_sleep_log(log, logging.WARNING, exc_info=True),
-        after=after_log(log, log_level=logging.ERROR),
+        before_sleep=before_sleep_log(_logger, logging.WARNING, exc_info=True),
+        after=after_log(_logger, log_level=logging.ERROR),
     ):
         with attempt:
             async with AsyncExitStack() as stack:
                 response = await stack.enter_async_context(session.get(url))
-                if response.status == 404:
+                if response.status == web.HTTPNotFound.status_code:
                     raise exceptions.InvalidDownloadLinkError(url)
-                if response.status > 299:
+                if response.status >= web.HTTPMultipleChoices.status_code:
```

Comment:
HTTPMultipleChoices?? is that a 299?

Reply:
Let me move this to a better constant name. Since this is a bit awkward.